src/components/tabs/README.md

```diff
@@ -0,0 +1,109 @@
+# UiTabs
+Tabs component that could be used in a SPA or even as simple enhancement for your DOM components
+
+## Usage
+```js
+import { UiTabs } from '@dreipol/vue-ui';
+import { UiTab } from '@dreipol/vue-ui';
+```
+
+## Tabs Props
+| Name | Type | Default | Description
+| --- | --- | ---| ---|
+|`selectedTab` | Number or null | null | Initially sets tab active
+|`hasAnimation` | boolean | false | Does tabs have transitions or not
+|`renderContentAlways` | boolean | false | Should visible and invisible content be always rendered to the dom
+
+## Tab Props
+| Name | Type | Default | Description
+| --- | --- | ---| ---|
+|`tabId` | Number or null | null | If set, it will match with selectedTab from Tabs and initially set the tab active
+|`hasAnimation` | boolean | null | Does the tab have transitions or not
+|`href` | string | empty | Setting href for the link 
```

Comment:
I'd be a little more precise here. Setting a href attribute (afaik and imho) changes the behaviour of a tab link significantly. Something like «If set, selecting the tab will redirect to the value of `href`. Any contents associated with this tab will *never* get rendered.»

To convert a tab button into a semantically correct link, you'd also have to accept and delegate at least the attributes `target` and `rel`. Not sure, whether this is out of scope for this PR.

Reply:
Changed as requested. I've added the attributes target and rel as props to tab. if href is not set, it will render a div.